example/secrets_manager.py

```diff
@@ -0,0 +1,14 @@
+import boto3
+import json
+
+def get_secret(secret_name):
+    # Create a Secrets Manager client
+    client = boto3.client('secretsmanager')
+
+    try:
+        get_secret_value_response = client.get_secret_value(SecretId=secret_name)
+    except Exception as e:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

We detected an `except` clause that is not necessary because it catches and re-throws an exception without doing anything with it. To call additional operations on the caught exception, consider wrapping it into a custom exception that adds more information or special handling.